### ISSUE TITLE
User suggestion: vector values

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/projectile-motion#151]
> I love the PhET sims for my AP and IB physics classes. I just used the new HTML5 Projectile motion sim, which is great. There's one more thing I'd love to see, and I think it could easily fit into this sim: Could you add a tool that could give the values of the velocity vectors (including the components of the velocity)? I think it could fit in the "Vectors" menu choice. We would use that a lot in my classes.

**Comment by andrea-phet** [comment]
My physics teacher uses the sim for her classes and she would really love to see this addition!

**Comment by ariel-phet** [comment]
@andrea-phet @arouinfar perhaps on the vectors screen we could put a tool the looks/behaves just like the trace tool but has a different background color and Gives V_tot V_x and V_y for each point?

My thought would be to remove the measuring tape on that screen and replace it with this tool (I think if you have "measuring tape learning goals" they could be gotten to on the other screens) and I would worry that we want to keep the play area as open as possible.

@andrea-phet - I assume that much like Time, Height, and Range, the velocity and components could be recorded for each point?

**Comment by andrea-phet** [comment]
Yes, velocity can be recorded. (In fact, it already is recorded just like time, height, etc., and just needs to be accessed).

**Comment by arouinfar** [comment]
> @andrea-phet @arouinfar perhaps on the vectors screen we could put a tool the looks/behaves just like the trace tool but has a different background color and Gives V_tot V_x and V_y for each point?

I really like this idea @ariel-phet!  The velocity trace tool would be useful beyond the goals of the Vectors screen, though. I think measuring V_x would be more useful on the Drag screen than the tape measure. So perhaps the tape measure could be on the Intro screen only?

In the [meeting notes for 2/9/2017](https://docs.google.com/document/d/19lCpjkR8Eefi5GLTBVSlHgtP0F7CGxPMDWz7N5AdvuI/edit#heading=h.4u43kqeg8yxm) we discussed the possibility of a vectors trace tool to show the vectors along the path and a snapshot tool to allow students to allow students to simultaneously compare the vectors at different locations on the path. We decided that these tools, while useful, were beyond the scope of 1.0. Not sure if this is too much feature creep, but perhaps we could include some of this functionality in the velocity trace tool. 

Since it sounds like the velocity is being already being recorded at each point, perhaps we could draw the velocity vectors at point being measured. We could also consider having two vector trace tools in the toolbox to allow simultaneous comparisons. Here's a rough mockup showing the velocity trace tool in action (color and labels TBD).
<img width="1025" alt="screen shot 2017-09-15 at 11 54 11 am" src="https://user-images.githubusercontent.com/8419308/30496576-0d38bcc2-9a0d-11e7-8b7a-4e4b00d23feb.png">

**Comment by ariel-phet** [comment]
@arouinfar I personally think I would just have the velocity trace tool only show values. For one, things will get a bit messy with the tool when the vectors overlap the tool. I also think the learning goals around the values don't necessarily need to show the vectors. To me the dynamic vector representation, and the ability to use the "step" as we already have it is much more compelling for the comparison of vectors at different points along the trajectory.

Seems like worth a quick discussion at design meeting with @kathy-phet

**Comment by arouinfar** [comment]
@ariel-phet I'd be find with only showing the values. I had remembered some discussion of features for 1.1+, and thought those features could possibly piggyback on the vector trace tool.

>  For one, things will get a bit messy with the tool when the vectors overlap the tool. 

Great point. Moving the probe to the right side of the tool will minimize vector overlap with the tool, but then becomes unusable for the first few points along the trajectory.
<img width="684" alt="screen shot 2017-09-18 at 4 30 47 pm" src="https://user-images.githubusercontent.com/8419308/30567687-c1950b20-9c8e-11e7-9df3-afa6af711bad.png">

> To me the dynamic vector representation, and the ability to use the "step" as we already have it is much more compelling for the comparison of vectors at different points along the trajectory.

While pause/step is powerful, it doesn't really facilitate simultaneous comparison of vectors along the trajectory. One could fire multiple projectiles at once (that's how I got two sets of vectors in the image above), but it doesn't really allow the user much control.

> Seems like worth a quick discussion at design meeting with @kathy-phet

Agreed! Unassigning myself for now.

**Comment by arouinfar** [comment]
When we discussed this feature last fall, there was some concern about such a tool giving away too much information. However, I think we may still want to consider some sort of velocity tool when this sim gets revisited. By then, we may have also had more teacher requests.

**Comment by zepumph** [comment]
Tagging @arouinfar because there is work being done in this sim again. Let me know if you want this implemented.

**Comment by arouinfar** [comment]
Looking at this issue with fresh eyes, I think I was overcomplicating this a bit.  It's already possible to qualitatively compare the velocity vectors along multiple points along the trajectory, as we allow up to three projectiles in the air at once.

![image](https://user-images.githubusercontent.com/8419308/67340628-25ca0800-f4eb-11e9-84c3-c3b98b093928.png)

The goal here is really to get some quantitative information about the velocity and its components. We could put the values on the vectors directly, but things would get crowded and would be nearly impossible to read unless the sim is paused. I think a velocity trace tool would be the most useful way to address this goal.

Here are a few mockups of what it could look like. Earlier mockups had the probe on the top or right side of the tool, but this placement would make it hard (or impossible) to measure certain points along the trajectory. We could also consider adding a title to the tool so the abbreviated labels are clearer. This has the added benefit of making the velocity trace tool a bit taller, so it would be easier to use with the other tool (tap tool to bring it to the front).
![image](https://user-images.githubusercontent.com/8419308/67347462-629ffa00-f4ff-11e9-9556-99cde3199aa8.png)
![image](https://user-images.githubusercontent.com/8419308/67404176-31134700-f570-11e9-9b47-895caf12c1a6.png)

Tagging for design meeting to check in with the team.

**Comment by zepumph** [comment]
@arouinfar I remember discussing this during design, but I forget what conclusion we came to. I thought that we at least decided that the above tool was not going to be created for the coming version. Is that right?

**Comment by arouinfar** [comment]
@zepumph the decision was that we were not going to add this feature to Projectile Motion, and the issue can be closed.

**Comment by arouinfar** [comment]
A similar request was made in #227 

> I was wondering if there are any options to see the V(x) and V(y) components VALUES on the projectile motion lab for any given MOMENT in time during the projectile path? This would help greatly in UAM equation verifications.

**Comment by oliver-phet** [comment]
Another similar request received into phethelp:

> In addition to showing the instantaneous velocity vector, can we get a measure of?  The could either be in the meter (the blue thing), or as a tick box option off to the right to include a label on the green vector itself. I use this as a predict & test mode of teaching.

**Comment by arouinfar** [comment]
Since there continue to be user requests about this feature, I'm reopening the issue so we can consider it during the next release.
